config/prod.exs

```diff
@@ -10,6 +10,9 @@ use Mix.Config
 # which you should run after static files are built and
 # before starting your production server.
 config :liveman, LivemanWeb.Endpoint,
+  http: [port: {:system, "PORT"}],
```

Comment:
Wouldn't that already be configured in `runtime.exs`?

https://github.com/nimblehq/liveman-demo-api/blob/d3dcfa4ecf0f3768a132a82e3d9e7d7e43211f22/config/runtime.exs#L14-L16

Reply:
I've tried without this config and it always return us the [Boot timeout error](https://devcenter.heroku.com/articles/error-codes#r10-boot-timeout).

![Screen Shot 2564-07-23 at 15 05 20](https://user-images.githubusercontent.com/6965195/126756624-342a0f74-e29a-465e-adc8-237987ce9641.png)

I saw there are facing this issue and fix by putting this in the config. I also try moving the same config to `runtime.exs` but it doesn't work. I doubt if someone face the same issue on Elixir IC. 🤔